### PR TITLE
Add book list with your styles

### PR DESCRIPTION
**Pull request by blacksam07**
## Summary
- Add book list
- Add detail.html file
- Add index.scss file

## Trello Card
[30-listado-de-libros](https://trello.com/c/ZLqZZdaA/30-listado-de-libros)

## Screenshots
image

- Screenshot updated
image

### CODE REVIEW

File: detail.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body class="container">
+  <div class="container-item badge">
+    <img class="book-cover" src="resources/book-cover.png" alt="book_cover">
```
Comment (CrisVega08) [inline detail.html]:
```suggestion
    <img class="book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 2]
```diff
@@ -8,26 +8,62 @@
...
-    </div>
-    <div class="item">
-      <span class="bold-text">Año de publicación:</span>
-      <span class="base-span">Año de publicación</span>
-    </div>
-  </div>
+<body class="list-container">
+  <a href="detail.html">
+    <div class="list-item">
+      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover">
```
Comment (CrisVega08) [inline index.html]:
```suggestion
      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 3]
```diff
@@ -8,26 +8,62 @@
...
+      <span class="item-bold-text">Título del libro</span>
+      <span class="item-text">Autor del libro</span>
+    </div>
+  </a>
+  <a href="detail.html">
+    <div class="list-item">
+      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover">
```
Comment (CrisVega08) [inline index.html]:
```suggestion
      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 4]
```diff
@@ -8,26 +8,62 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 5]
```diff
@@ -8,26 +8,62 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 6]
```diff
@@ -8,26 +8,62 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 7]
```diff
@@ -8,26 +8,62 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 8]
```diff
@@ -8,26 +8,62 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 9]
```diff
@@ -8,26 +8,62 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html [hunk 10]
```diff
@@ -8,26 +8,62 @@
...
+<body class="list-container">
+  <a href="detail.html">
+    <div class="list-item">
+      <img class="item-book-cover" src="resources/book-cover.png" alt="book_cover">
+      <span class="item-bold-text">Título del libro</span>
+      <span class="item-text">Autor del libro</span>
+    </div>
+  </a>
+  <a href="detail.html">
+    <div class="list-item">
```
Comment (betoobernal) [inline index.html:20]: i could think that you might use the `a` and remove the `div` 🤔

File: index.html [hunk 11]
```diff
@@ -8,26 +8,62 @@
...
-      <span class="bold-text">Editorial:</span>
-      <span class="base-span">Nombre de la Editorial</span>
-    </div>
-    <div class="item">
-      <span class="bold-text">Año de publicación:</span>
-      <span class="base-span">Año de publicación</span>
-    </div>
-  </div>
+<body class="list-container">
+  <a href="detail.html">
```
Comment (CrisVega08) [inline index.html]: After @beto @yo decided to use only the <a href="detail.html"> in the image, what do you think?

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: index.html (hunk 11 above)
Comment (blacksam07) [reply index.html]: it's ok, no problem I make these changes

**Review by betoobernal — APPROVED** [review]

File: scss/index.scss [hunk 12]
```diff
@@ -0,0 +1,32 @@
+.list-container {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  width: 1000px;
```
Comment (CrisVega08) [inline scss/index.scss]: I suggest use width: 100% ;
and max-width 1000px;

File: scss/index.scss [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+.list-item {
+  box-shadow: 4px 4px 10px 0 $black-lighter-5;
+  height: 300px;
+  margin-top: 35px;
+  margin-left: 35px;
```
Comment (CrisVega08) [inline scss/index.scss:13]: sort properties

File: scss/index.scss [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+  padding: 25px;
+  width: 200px;
+}
+
+.item-book-cover {
+  height: 200px;
+  width: 150px;
+  background: center / cover no-repeat url('resources/book-cover.png');
```
Comment (CrisVega08) [inline scss/index.scss]: sort properties

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

**Review by betoobernal — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**